src/main/java/org/apache/ibatis/parsing/XNode.java

```diff
@@ -130,71 +130,47 @@ public String getStringBody() {
   }
 
   public String getStringBody(String def) {
-    if (body == null) {
-      return def;
-    } else {
-      return body;
-    }
+    return body == null ? def : body;
   }
 
   public Boolean getBooleanBody() {
     return getBooleanBody(null);
   }
 
   public Boolean getBooleanBody(Boolean def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Boolean.valueOf(body);
-    }
+    return body == null ? def : Boolean.valueOf(body);
   }
 
   public Integer getIntBody() {
     return getIntBody(null);
   }
 
   public Integer getIntBody(Integer def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Integer.parseInt(body);
-    }
+    return body == null ? def : Integer.valueOf(body);
   }
 
   public Long getLongBody() {
     return getLongBody(null);
   }
 
   public Long getLongBody(Long def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Long.parseLong(body);
-    }
+    return body == null ? def : Long.parseLong(body);
```

Comment:
You use `valueOf()` in `getLongAttribute()`.
Either `valueOf` or `parseLong` looks OK to me, but we should use the same one in both methods.